Update easy-way-of-sharing-files.md (#2581)
enkidevs/curriculum: linux/user-and-file-management/file-management-tips-ii/easy-way-of-sharing-files.md

`woof` (Web Offer One File) is a script and will run on any machine with *Python* installed.

> You can check download it from the link in the **Learn More** section.

Reviewer: what do you think about this?

```suggestion
To download it, check the **Learn More** section.
```

Author: Uh, what happened. That was not what I wrote haha.

It was supposed to say, `you can check it out at the **Learn More** section.`

Yeah I like that I'll commit it
